shared/src/packages/projects/libs/types/get-projects-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type GetProjectsItemResponseDto } from './get-projects-item-response-dto.type.js';
+
+type GetProjectsResponseDto = {
```

Comment:
```suggestion
type ProjectGetAllResponseDto = {
```
check how it looks in other packages (ex. users)